File: Server side/TCPServer_simple/Restaurant.java
import java.io.Serializable;

public class Restaurant implements Serializable{

	private String name;
	private String raw_ranking;
	private String address;
	private String photo;
	private String userEmail;
	

	public Restaurant(String name, String raw_ranking, String address, String photo, String userEmail) {
		
	}

	public String getName() {
		return name;
	}
	public String getraw_ranking() {
		return raw_ranking;
	}
	public String getaddress() {
		return address;
	}
	public String getphoto() {
		return photo;
	}
	public String getuserEmail() {
		return userEmail;
	}
	
	public void setName(String name) {
		this.name=name;
	}
	public void setraw_ranking(String raw_ranking) {
		this.raw_ranking=raw_ranking;
	}
	public void setaddress(String address) {
		this.address=address;
	}
	public void setphoto(String photo) {
		this.photo=photo;
	}
	public void setuserEmail(String userEmail) {
		this.userEmail=userEmail;
	}
	
	
}

File: Client side/app/src/main/java/com/example/nika/android2/models/Restaurant.kt
package com.example.nika.android2.models

import java.io.Serializable

/**
 * Created by EP on 18/07/2017.
 */
class Restaurant : Serializable {
    var name: String
        private set
    var raw_ranking: String
        private set
    lateinit var address: String
        private set
    lateinit var photo: String
        private set
    lateinit var userEmail: String
    var id: Long=0
    //var min_p: Long=0


    constructor(name: String, raw_ranking: String) {
        this.name = name
        this.raw_ranking = raw_ranking
    }

    constructor(name: String,  raw_ranking: String, address: String, photo: String,id:Long) {
        this.name = name
        this.raw_ranking = raw_ranking
        this.address = address
        this.photo = photo
        this.id = id
    }
    constructor(name: String,  raw_ranking: String, address: String, photo: String,id:Long,userEmail:String) {
        this.name = name
        this.raw_ranking = raw_ranking
        this.address = address
        this.photo = photo
        this.userEmail = userEmail;
        this.id = id
    }

}